src/OtpInput/__snapshots__/OtpInput.test.tsx.snap

```diff
@@ -396,6 +396,7 @@ exports[`OtpInput UI should render correctly 1`] = `
     </View>
   </View>
   <TextInput
+    autoComplete="sms-otp"
```

Comment:
What about setting it to "one-time-code" because from the docs I see that "sms-otp" is Android specific prop value? Thank you for such a nice PR!

Reply:
No worries! I was getting a type error because react-native is a few versions behind. I've bumped, just to allow for this change and set it to `one-time-code` without having a prop